tailwind.config.js

```diff
@@ -41,10 +41,15 @@ module.exports = {
         },
         primary: {
           DEFAULT: "#d8b36d",
+          light: "#E8D1A7",
```

Comment:
For the `gray` color we used 700. Can we rename "light" and "lighter" to something similar? That way things are a bit more consistent.

Reply:
Sure, thoughI feel like it's a lot more confusing/"random" to read. We should probably never have more than a handful of shades for each given color anyway (imho).

Are the shade numbers based on anything? I know a bigger number means darker but I have no idea which values to pick here haha.

Reply:
Right. So the reason I like codes like 100-900, is because that's what Tailwind uses. However, maybe it makes more sense to just stick to light/lighter/dark/darker! In which case, can you update the gray color to use the same naming convention?

Reply:
Alright so I used `DEFAULT` for the other ones as I figured if we don't have other values yet it probably means they're the only values for the given colors of our theme. Not 100% sure though so feel free to change them again!